src/storm-dft/generator/DftNextStateGenerator.cpp

```diff
@@ -77,7 +77,7 @@ storm::generator::StateBehavior<ValueType, StateType> DftNextStateGenerator<Valu
     // Prepare the result, in case we return early.
     storm::generator::StateBehavior<ValueType, StateType> result;
 
-    STORM_LOG_TRACE("Currently failable: " << state->getFailableElements().getCurrentlyFailableString());
+    STORM_LOG_TRACE("Currently failable: " << state->getFailableElements().getCurrentlyFailableString(!exploreDependencies));
     // TODO remove exploreDependencies
```

Comment:
Does this PR affect this TODO?

Reply:
I removed the TODO because I need the variable `exploreDependencies` in the case where dependency failures cannot occur to due restrictors and we have to explore the BE failures instead.